.github/workflows/ci-dotnet.yml

```diff
@@ -44,6 +42,6 @@ jobs:
       run: |
         bazel build //dotnet/test/common:${{ matrix.browser }}
         $env:ACTIVE_DRIVER_CONFIG = '${{ matrix.driver }}'
-        $env:DRIVER_SERVICE_LOCATION = '${{ matrix.location }}'
+        rm C:/SeleniumWebDrivers -r
```

Comment:
Here `C:/SeleniumWebDrivers` is hardcoded and OS specific (will not work on Linux). Should we make it OS agnostic? Easy fix, but then we will see 3 entries in yaml, like `remove chrome`, `remove gecko`.

Reply:
Yes, we should. Will fix it.

Reply:
We can do conditionals based on which runner is being used.

Reply:
I tried to use `rm "$env:ChromeWebDriver" "$env:EdgeWebDriver" "$env:GeckoWebDriver" -r`

Reply:
Added `-v` flag to see what was being deleted.

```
VERBOSE: Performing the operation "Remove Directory" on target "C:\SeleniumWebDrivers\ChromeDriver".
VERBOSE: Performing the operation "Remove File" on target "C:\SeleniumWebDrivers\ChromeDriver\chromedriver.exe".
VERBOSE: Performing the operation "Remove File" on target "C:\SeleniumWebDrivers\ChromeDriver\LICENSE.chromedriver".
VERBOSE: Performing the operation "Remove File" on target "C:\SeleniumWebDrivers\ChromeDriver\versioninfo.txt".
VERBOSE: Performing the operation "Remove Directory" on target "C:\SeleniumWebDrivers\EdgeDriver".
VERBOSE: Performing the operation "Remove Directory" on target "C:\SeleniumWebDrivers\EdgeDriver\Driver_Notes".
VERBOSE: Performing the operation "Remove File" on target "C:\SeleniumWebDrivers\EdgeDriver\Driver_Notes\credits.html".
VERBOSE: Performing the operation "Remove File" on target "C:\SeleniumWebDrivers\EdgeDriver\Driver_Notes\EULA".
VERBOSE: Performing the operation "Remove File" on target "C:\SeleniumWebDrivers\EdgeDriver\Driver_Notes\LICENSE".
VERBOSE: Performing the operation "Remove File" on target "C:\SeleniumWebDrivers\EdgeDriver\msedgedriver.exe".
VERBOSE: Performing the operation "Remove File" on target "C:\SeleniumWebDrivers\EdgeDriver\versioninfo.txt".
VERBOSE: Performing the operation "Remove Directory" on target "C:\SeleniumWebDrivers\GeckoDriver".
VERBOSE: Performing the operation "Remove File" on target "C:\SeleniumWebDrivers\GeckoDriver\geckodriver.exe".
VERBOSE: Performing the operation "Remove File" on target "C:\SeleniumWebDrivers\GeckoDriver\versioninfo.txt".
```